core/workspace.js

```diff
@@ -170,6 +181,50 @@ Blockly.Workspace.prototype.getTopBlocks = function(ordered) {
   return blocks;
 };
 
+/**
+ * Add a comment to the list of top comments.
+ * @param {!Blockly.WorkspaceComment} comment comment to add.
+ * @package
+ */
+Blockly.Workspace.prototype.addTopComment = function(comment) {
+  this.topComments_.push(comment);
+};
+
+/**
+ * Remove a comment from the list of top comments.
+ * @param {!Blockly.WorkspaceComment} comment comment to remove.
+ * @package
+ */
+Blockly.Workspace.prototype.removeTopComment = function(comment) {
```

Comment:
I may have asked this before, but why does removeTopComment not remove the comment from the DB too?

Reply:
Following the blocks model unnecessarily--blocks can be non-top-level while still being on the workspace.

That may eventually be true for these comments, if block comments merge with workspace comments.  But for now it makes sense for me to make addTopComment add it to the db and removeTopComment remove it from the db.  I'll do that.

Reply:
Done.